webpack/components/extensions/HostDetails/Tabs/__tests__/packagesTab.test.js

```diff
@@ -255,6 +255,98 @@ test('Can upgrade a package via customized remote execution', async (done) => {
   assertNockRequest(statusScope, done);
 });
 
+test('Can remove a package via remote execution', async (done) => {
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+
+  const scope = nockInstance
+    .get(hostPackages)
+    .query(defaultQuery)
+    .reply(200, { ...mockPackagesData, results: [firstPackage] });
+
+  const upgradeScope = nockInstance
+    .post(jobInvocations, {
+      job_invocation: {
+        inputs: {
+          package: firstPackage.name,
+        },
+        search_query: `name ^ (${hostname})`,
+        feature: REX_FEATURES.KATELLO_PACKAGE_REMOVE,
+      },
+    })
+    .reply(201);
+
+  const {
+    getByText,
+    getAllByText,
+    getByRole,
+    getByLabelText,
+  } = renderWithRedux(<PackagesTab />, renderOptions());
+
+  await patientlyWaitFor(() => expect(getAllByText(firstPackage.name)[0]).toBeInTheDocument());
+
+  getByRole('checkbox', { name: 'Select row 0' }).click();
+  expect(getByLabelText('Select row 0').checked).toEqual(true);
```

Comment:
This is the test that uses the table row kebab, right? In that case we don't need to check the box first.